webapp/apifuncs.py

```diff
@@ -202,6 +202,31 @@ def add_todays_results(result1, result2, playercode, fullname, pos, clubname):
     return result
 
 
```

Comment:
Just a question before merging as I just really realized this. The stat_translation is set between two functions, not inside of either one. I will need to update this to include the pitching stat translation entries when I make that routine more pythonic. Is this common/ok to leave something of this nature outside of a function?